.github/workflows/build-and-test.yaml

```diff
@@ -44,3 +44,10 @@ jobs:
         run: python -m pip install tox
       - name: Run integration tests
         run: tox -e integration
+      - name: Setup operator environment
+        uses: charmed-kubernetes/actions-operator@main
```

Comment:
Why is charmed-kubernetes need here? Does it add much time to the integration tests run?

Reply:
This is just a standard action that sets up charmcraft/juju/lxd etc., so not necesarrily deploy k8s in this instance.

My only question is whether it's needed - could this be covered with more simple functional tests using LXD like the other libraries?

Reply:
The reason I added this step is because I need a mini-Juju cloud to test that the `juju-systemd-notices` daemon properly feeds `ServiceStartEvent`|`ServiceStopEvent` events into a test charm and that the test charm properly handles the events. It does add additional overhead to the integration tests, but I need a Juju environment to ensure that the charm lib is working as expected in practice.